types/react-big-calendar/react-big-calendar-tests.tsx

```diff
@@ -198,7 +198,7 @@ class CalendarResource {
                         const end = slotInfo.end;
                         return true;
                     }}
-                    dayLayoutAlgorithm="overlap"
+                    dayLayoutAlgorithm={customLayoutAlgorithm}
```

Comment:
This is undefined at this point in the code. Could you move the actual `customLayoutAlgorithm` up so that the variable is declared and used in the same scope?

Reply:
I debated this point before committing it, I put it with the rest of the data functions rather than starting them in two places. Do we think it would be better to move all of these or leave it as is?

Reply:
They should live close to another and not across scopes. I'm honestly a bit surprised this even works. `customLayoutAlgorithm` is declared after usage so you could potentially call render and get an undefined ´customLayoutAlgorithm` because of "temporal dead zone". Not relevant for static tests but we want to them to be as close as possible to actual usage.

Reply:
Is the verdict here to move all of the functions passed by reference to above where they are used or just this one. I personally feel keeping them together is far more useful than splitting them, which would logically put them all at the top of the file. If so, would this be better in a secondary PR to keep the changes clear?